framer/Components/SliderComponent.coffee

```diff
@@ -138,6 +178,21 @@ class exports.SliderComponent extends Layer
 			@knob.height = @_knobSize
 			@_updateFrame()
 
+	@define "tapSize",
+		get: -> @_tapSize
+
+		set: (value) ->
+
+			@_tapSize = value
+
+			@knobOverlay.props =
+				width: @knobSize or 30 + @_tapSize
+				height:	@knobSize or 30 + @_tapSize
+
+			@sliderOverlay.props =
+				width: @width + @_tapSize
+				height:	@height + @_tapSize
+
```

Comment:
@benjaminnathan @koenbok Let's open up the discussion on the naming here: `tapSize`. Can we find a better term that describes by how many pixels the track- and thumb area get extended by for hit testing? I can think of:

- hitAreaExtension
- hitAreaBorderSize
- hitArea
- extendedHitAreaSize
- hitAreaMargin